core/src/main/scala/org/apache/spark/SecurityManager.scala

```diff
@@ -87,10 +87,14 @@ private[spark] class SecurityManager(
   private var secretKey: String = _
   logInfo("SecurityManager: authentication " + (if (authOn) "enabled" else "disabled") +
     "; ui acls " + (if (aclsOn) "enabled" else "disabled") +
-    "; users  with view permissions: " + viewAcls.toString() +
-    "; groups with view permissions: " + viewAclsGroups.toString() +
-    "; users  with modify permissions: " + modifyAcls.toString() +
-    "; groups with modify permissions: " + modifyAclsGroups.toString())
+    "; users  with view permissions: " +
+    (if (viewAcls.nonEmpty) viewAcls.mkString(",") else "EMPTY") +
```

Comment:
nit: In `mkString(",")`,  add an empty space after `,` ?

Reply:
OK :)